src/client/client.test.js

```diff
@@ -111,6 +111,25 @@ describe.each(['httpClient', 'async httpClient'])('%s', (description) => {
     );
   });
 
+  // This test case will test batch output of graffy
+  test('store query batching', async () => {
+    await Promise.all([
+      store.read({ demo: 1 }),
+      store.read({ anotherDemo: 1 }),
+      store.read({ demo: 1 }),
+      store.read({ anotherDemo: 1 }),
+    ]);
+    expect(getOptions).toHaveBeenCalled();
+    expect(fetch).toHaveBeenCalledWith(
```

Comment:
Let's add one more assertion,

```js
expect(fetch).toHaveBeenCalledTimes(1);
```

To verify that we are not accidentally calling fetch 4 times with the combined query.